xplique/attributions/image_object_detector_explainer.py

```diff
@@ -0,0 +1,280 @@
+"""
+Module related to Object detector method

```

Comment:
Do we need a proper folder to store the bouding box explainers or 1 file is enough ? @dejeanph what do you think (in the future I mean)?

Reply:
Each explained bounding box is drawn in a single image. If x bounding boxes are detected, x image results have to be save.
So a folder is usefull. By default, the algorithm can add the id of the bounding box as a suffix to the image filename.

Reply:
I mean a folder in the attribution folder, instead of `xplique/attributions/bbox_explainer.py` => `xplique/attributions/bbox/explainer.py` ? 
But I agree, let's start with that